signature-react-app/src/App.tsx

```diff
@@ -185,6 +206,31 @@ function App() {
     }
   };
 
+  const downloadHtmlFile = () => {
+    const htmlSignature = ReactDOMServer.renderToStaticMarkup(
+      <Signature
+        logo={state.logo}
+        fullName={state.fullName}
+        credentials={state.credentials}
+        title={state.title}
+        phone={state.phone}
+        mobile={state.mobile}
+        calendlyLink={state.calendlyLink}
+      />
+    );
+    const blob = new Blob([htmlSignature]);
+    const fileDownloadUrl = URL.createObjectURL(blob);
+    const link = document.createElement("a");
+    link.href = fileDownloadUrl;
+    link.setAttribute(
+      "download",
+      `${state.fullName.split(" ").join("-")}-signature.htm`
```

Comment:
We need to match the output (file naming) of the bulk signature generator, right?

Chad may be scripting the file placement by staff name. We can at least be consistent.

Reply:
<img width="1290" alt="Screen Shot 2022-12-30 at 8 50 12 AM" src="https://user-images.githubusercontent.com/75748078/210083059-0129ed42-1bb3-4e01-bc39-bcdb37dafa09.png">